### DISCUSSION TITLE
Add "if _" tooltip to pp in score results screen

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#25417] [Ideas]
Currently, I can think of: "if FC", "if % (accuracy)", and "if NM":
- FC/% - Good to visualize how close you are to FC/%, and compare with it.
- NM - Good to compare btwn different mod combinations, and to guide newer players (ex: they see that relax gives significantly less pp, so they'll avoid it, or they see that DT gives significantly more pp, so they'll gravitate towards it).

**Comment by bdach** [comment]
I know people do this with scoreposts but I also kinda don't like it. Just hit the score if you want to see how much PP it gives.

> ex: they see that relax gives significantly less pp, so they'll avoid it, or they see that DT gives significantly more pp, so they'll gravitate towards it

I don't know that we want to encourage pp farm even harder than it already is at this point.

**Reply by Rekunan** [reply]
I don't see a bad thing with encouraging it. Def could be better than now, but it at least naturally guides players to pushing their skillcap instead of staying stagnant.

**Reply by marioman63** [reply]
From my experience playing with others or talking with other players, I find the farming behaviour very toxic and a form of "metagaming" which, if you enjoy, more power to you, but I am very happy the devs are trying not to encourage such behaviour. Everyone progresses in their own way and displaying such stats may convey the message that the only way to improve is in one specific way which could discourage some players from playing entirely.

**Reply by Rekunan** [reply]
I think that's very anecdotal experience, considering how the players I talk to feel and/or think the opposite.
It depends on how the player interprets the information, rather than it being inherently toxic. It may have implications, but at the end of the day, it's a useful number that the player can interpret one way or another.

**Comment by peppy** [comment]
Can we just show PP as a x/MAX like other components on the results screen?

**Reply by bdach** [reply]
We already show maximum PP on the extended statistics screen. This is asking for something slightly different, as in "show me the pp points I would get if I didn't break combo at any point". See what https://github.com/ppy/osu/pull/25624 does, because it's actually trying to implement this; it fills the misses with _something_ to simulate "an FC".

**Reply by peppy** [reply]
right. i dunno.
